generate_questions.py

```diff
@@ -0,0 +1,74 @@
+import os
+from pathlib import Path
+
+# Function to load LOs and K-levels from a text file
+def load_learning_objectives(file_path):
+    data = []
+    with open(file_path, 'r', encoding='utf-8') as file:
+        for line in file:
+            line = line.strip()
+            if line:
+                # Split by comma and strip any extra spaces from each part
+                lo, k_level = [part.strip() for part in line.split(',')]
+                data.append((lo, k_level))
+    return data
+
+
+# Load the LOs and K-levels from the file 'learning-objectives'
+file_path = 'learning-objectives'  # Provide the correct path to your file
```

Comment:
I would enable users to define path to the source CSV file as an optional script parameter. But if not found, default to `learning-objectives.csv`

Reply:
updated